compiler_opt/es/blackbox_optimizers.py

```diff
@@ -1095,7 +1113,7 @@ def update_state(self, evaluation_stats):
           for a, b in zip(mean_squares_state_vector, self.mean_state_vector)
       ]
 
-  def set_state(self, state):
+  def set_state(self, state: Union[List, npt.NDArray]) -> None:
```

Comment:
Union[Sequence[float], npt.NDArray]

guessing these are floats, right?  same in e.g. ,L1092 and other places where you take as input a List.

Reply:
This has been updated now